# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/mc/net-ftplink-key.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/main.c.orig	2003-04-30 23:03:39.000000000 +0300
+++ ./src/main.c	2003-06-21 14:15:58.000000000 +0300
@@ -1801,7 +1801,9 @@
     { 'q',          quick_cmd_no_menu },
     { 'h',          add2hotlist_cmd },
     { '!',          external_panelize },
+#ifdef USE_NETCODE			    
     { 'f',	    ftplink_cmd },
+#endif
 #ifdef WITH_BACKGROUND
     { 'j',          jobs_cmd },
 #endif
